drivers/overlay/overlay.go

```diff
@@ -726,6 +726,24 @@ func (d *Driver) LookupAdditionalLayer(dgst digest.Digest, ref string) (graphdri
 	}, nil
 }
 
+// LookupAdditionalLayerByID looks up additional layer store by the specified
+// ID and returns an object representing that layer.
+// This API is experimental and can be changed without bumping the major version number.
```

Comment:
The comment is great, but it won't age well.   Perhaps a TODO: to remove the comment once it's no longer experimental.